backend/driver.pm

```diff
@@ -98,7 +98,7 @@ sub start {
 
             $self->{backend}->run(fileno($process->channel_in), fileno($process->channel_out));
             _exit(0);
-    })->blocking_stop(1)->separate_err(0)->subreaper(1)->start;
+    }, sleeptime_during_kill => .1)->blocking_stop(1)->separate_err(0)->subreaper(1)->start;
```

Comment:
Interesting find. This is [not even documented](https://metacpan.org/pod/Mojo::IOLoop::ReadWriteProcess). I guess you spotted its use in Mojo.

Reply:
yes, unfortunately. Only https://metacpan.org/pod/Mojo::IOLoop::ReadWriteProcess#kill_sleeptime is documented.